Document/0x04f-Testing-Network-Communication.md

```diff
@@ -70,75 +73,34 @@ Wireshark offers a GUI and is more straightforward if you are not used to the co
 
 **Network Setup**
 
-To be able to get a man-in-the-middle position your machine should be in the same wireless network as the mobile phone and the gateway it communicates to. Once this is done you need the following information:
-
-- IP address of mobile phone
-- IP address of gateway
+To be able to get a man-in-the-middle position your machine should be in the same wireless network as the mobile phone and the gateway it communicates to. Once this is done you need the IP address of mobile phone.
 
-#### ARP Poisoning with Ettercap
-
-Start ettercap with the following command and replace the first IP addresses with the network gateway in the wireless network and the second one with the one of your mobile device.
-
-```shell
-$ sudo ettercap -T -i en0 -M arp:remote /192.168.0.1// /192.168.0.105//
-```
+#### ARP Poisoning with bettercap
 
-On the mobile phone start the browser and navigate to example.com, you should see output like the following:
+Start your preferred network analyzer tool first, then start bettercap with the following command and replace the IP address 192.168.0.103 with the target you want to execute the MITM attack against.
 
 ```shell
-ettercap 0.8.2 copyright 2001-2015 Ettercap Development Team
-
-Listening on:
-   en0 -> AC:BC:32:81:45:05
-	  192.168.0.105/255.255.255.0
-	  fe80::c2a:e80c:5108:f4d3/64
-
-SSL dissection needs a valid 'redir_command_on' script in the etter.conf file
-Privileges dropped to EUID 65534 EGID 65534...
-
-  33 plugins
-  42 protocol dissectors
-  57 ports monitored
-20388 mac vendor fingerprint
-1766 tcp OS fingerprint
-2182 known services
-
-Scanning for merged targets (2 hosts)...
-
-* |=========================================>| 100.00 %
-
-2 hosts added to the hosts list...
-
-ARP poisoning victims:
-
- GROUP 1 : 192.168.0.1 F8:E9:03:C7:D5:10
+$ sudo bettercap -eval "set arp.spoof.targets 192.168.0.103; arp.spoof on; set arp.spoof.internal true; set arp.spoof.fullduplex true;"
+bettercap v2.22 (built for darwin amd64 with go1.12.1) [type 'help' for a list of commands]
 
- GROUP 2 : 192.168.0.102 20:82:C0:DE:8F:09
-Starting Unified sniffing...
-
-Text only Interface activated...
-Hit 'h' for inline help
+[19:21:39] [sys.log] [inf] arp.spoof enabling forwarding
+[19:21:39] [sys.log] [inf] arp.spoof arp spoofer started, probing 1 targets.
+192.168.0.0/24 > 192.168.0.100  »
+```
```

Comment:
given this, i am fine with closing https://github.com/OWASP/owasp-mstg/issues/1043

Reply:
👍